pkgs/development/tools/glpaper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchhg
+, meson, ninja
+, pkg-config
+, wlroots, wayland, wayland-protocols
+, pixman, libxkbcommon
+,libX11
+, libGL, libglvnd
+}:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
```

Comment:
This should be `unstable-$date` as per https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Done.